### ISSUE TITLE
🐛 Warning message when setting Japanese tags

### ISSUE THREAD

**Issue by taromn** [DataDog/browser-sdk#2945]
**Describe the bug**

[Browser Log Collection(NPM)](https://docs.datadoghq.com/logs/log_collection/javascript/)

When setting Japanese service name, logs can be sent to Datadog, but SDK returns warning.

```
import { datadogLogs } from '@datadog/browser-logs'

datadogLogs.init({
  clientToken: 'xxxxxx',
  site: 'datadoghq.com',
  forwardErrorsToLogs: true,
  sessionSampleRate: 100,
  service: 'てすと',
})
```

↓ browser console
![Screenshot 2024-08-22 at 9 35 19](https://github.com/user-attachments/assets/da5580a2-ab18-42ea-a91c-6e0f47c6c193)

↓ log
![Screenshot 2024-08-22 at 10 20 31](https://github.com/user-attachments/assets/7e295f3b-ae24-4149-be85-3cba682bf4a9)

https://github.com/DataDog/browser-sdk/blob/9953ec5ddb5a4b55a02f9ad54d482f8ee59b8ec4/packages/core/src/domain/configuration/tags.ts#L36

It seems Japanese tags are supported.
https://docs.datadoghq.com/getting_started/tagging/#define-tags
> 2.Tags can be up to 200 characters long and support Unicode letters (which includes most character sets, including languages such as Japanese).

**To Reproduce**
- set Japanese tags

**Expected behavior**
SDK doesn't return the warning message just because the tag is Japanese.

**Comment by amortemousque** [comment]
Hello @taromn,
Thanks for reporting the issue! I'll let you know when it is fixed.

**Comment by amortemousque** [comment]
Hello @taromn,

We released a fix in the browser SDK v5.26.0.

**Comment by taromn** [comment]
thank you!
